rfc/rfc-0003-developer-utils.md

```diff
@@ -0,0 +1,74 @@
+# RFC-#: A pod with utilities for better developer experience
+
+**Authors:** Swapnil Bawaskar
+
+**Status:**
+
+**Pull Request URL:** [#1362](https://github.com/projectriff/riff/pull/1362)
+
+**Superseded by:** N/A
+
+**Supersedes:** [RFC 0001](https://github.com/projectriff/riff/pull/1359)
```

Comment:
I guess this PR should also update the "Superseded by" section of RFC 0001?

Reply:
Superseded by for rfc-0001 was updated when I updated the status to "rejected" and merged it: https://github.com/projectriff/riff/pull/1359.